### ISSUE TITLE
ERROR:process_info.cc(632)]  error while launching browser

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#11627]
### Description
When launching browser from console shows error

### Steps to Reproduce

  1. Install 0.19.61
  2. Keep browsing until you encounter browser crash
  3. Launch from console after the crash shows error 
  4. Close all tabs, clear cache and relaunch browser, similar error but shows different memory address
  5. Upgrade to 0.19.62, launch from console, similar error message

**Actual result:**

```
$ An uncaught exception occurred in the main process Uncaught Exception:
undefined: undefined
[1023/081203.244:ERROR:process_info.cc(632)] range at 0x5a43492c00000000, size 0x207 fully unreadable
[1023/081203.244:ERROR:process_info.cc(632)] range at 0x5a43494c00000000, size 0x207 fully unreadable
[1023/081203.244:ERROR:process_info.cc(632)] range at 0x0, size 0x207 fully unreadable
Waiting 60 seconds for process to load
Error: unrecognized flag --icu_case_mapping
Try --help for option
```

**Expected result:**
Should not show the error

**Reproduces how often:** [What percentage of the time does it reproduce?]
100% after the initial crash
### Brave Version

**about:brave info:**

Brave| 0.19.61
-- | --
rev| c8e042c
Muon| 4.5.6

Brave | 0.19.62
-- | --
rev | fe3f350
Muon | 4.5.7

**Reproducible on current live release:**

No
### Additional Information

#11604
Encountered the crash after a while of enabling honey extension. No crash after disabling it.

**Comment by srirambv** [comment]
```
$ An uncaught exception occurred in the main process Uncaught Exception:
undefined: undefined
[1023/084200.771:ERROR:process_info.cc(632)] range at 0xee45492c00000000, size 0x23f fully unreadable
[1023/084200.771:ERROR:process_info.cc(632)] range at 0xee45494c00000000, size 0x23f fully unreadable
[1023/084200.771:ERROR:process_info.cc(632)] range at 0x0, size 0x23f fully unreadable
Waiting 60 seconds for process to load
Error: unrecognized flag --icu_case_mapping
Try --help for options
[7196:7532:1023/084203.580:ERROR:CONSOLE(52003)] "(node) warning: possible EventEmitter memory leak detected. %d listeners added. Use emitter.setMaxListeners() to increase limit.", source: chrome://brave/C:/Users/Sriram%20BV/AppData/Local/brave/app-0.19.62/resources/app.asar/app/extensions/brave/gen/app.entry.js (52003)
[7196:7532:1023/084203.580:ERROR:CONSOLE(52003)] "(node) warning: possible EventEmitter memory leak detected. %d listeners added. Use emitter.setMaxListeners() to increase limit.", source: chrome://brave/C:/Users/Sriram%20BV/AppData/Local/brave/app-0.19.62/resources/app.asar/app/extensions/brave/gen/app.entry.js (52003)
```
Shows similar error even for a new profile.

**Comment by srirambv** [comment]
Able to reproduce the error on new profile from 0.19.57 onward.  Doesn't happen on 0.19.54 
cc: @bbondy

**Comment by bridiver** [comment]
I can't repro on mac

**Comment by NejcZdovc** [comment]
@bridiver I think this is on Windows

**Comment by bridiver** [comment]
ok, I'll try there

**Comment by bridiver** [comment]
@srirambv "Keep browsing until you encounter browser crash" - shouldn't happen anymore with the latest preview. Of course there might still be a crash at some point, but it seems like you must not be testing with muon 4.5.7?

**Comment by srirambv** [comment]
@bridiver thats correct crash happened on muon 4.5.6 with 0.19.61. But getting that error on each clean install from 0.19.57 onwards

**Comment by darkdh** [comment]
@srirambv the error might not be related to the crash

**Comment by alexwykoff** [comment]
@srirambv please check vs latest build when you have a moment

**Comment by srirambv** [comment]
Still reproducible on 0.19.63. Unable to attach recording. Shared on slack.

**Comment by darkdh** [comment]
Can you send me or @bridiver the latest modified file in your brave_profile folder/Crashpad/completed just after crash happens?

**Comment by darkdh** [comment]
the latest dump was generated by 0.19.62. I didn't see any dump generated by 0.19.63 from your dump files

**Comment by darkdh** [comment]
And your old dump(4.5.6) I inspected is the ukm crash which is fixed in muon 4.5.7

**Comment by bbondy** [comment]
This seems to be logging only so removing P1, milestone, and crash labels. Please let me know if that's not correct.
